_data/faq.yml

```diff
@@ -2373,107 +2373,6 @@
                 Nach dieser Änderung sollte dein Internet-Browser wieder auswählbar sein, wenn du "Öffne in Browser" aus c:geo wählst.
 
-- id: calendar-plugin
-  title:
-      en: Calendar plugin
-      de: Kalender-Erweiterung
-  items:
-      - id: plugin-calendar
-        en:
-            title: What does the calendar plugin do?
-            content: |
-                It allows adding event caches to your personal calendar.
-                As it is a plugin it will only work if also c:geo itself is installed on your device.
-                The plugin is only needed for devices running Android 2.3 or 3.x. On all newer Android versions the functions described below do not require the plugin.
-        de:
-            title: Wozu dient die Kalender-Erweiterung?
-            content: |
-                Sie ermöglicht es Event-Caches zu deinen persönlichen Kalender hinzuzufügen.
-                Da es eine Erweiterung ist, funktioniert sie nur wenn auch c:geo auf deinem Gerät installiert ist.
-                Die Erweiterung wird nur für Geräte mit Android-Version 2.3 oder 3.x benötigt. Für alle neueren Android-Versionen wird die Erweiterung zur Nutzung der unten beschriebenen Funktionen nicht benötigt.
-
-      - id: plugin-calendar-howto
-        en:
-            title: How do I add an event to my calendar?
-            content: |
-                When you open an event cache in c:geo select\\
-                Menu → Add to Calendar.
-
-                A new appointment for the event will be created in your calendar.
-        de:
-            title: Wie füge ich ein Event zu meinem Kalender hinzu?
-            content: |
-                Nachdem du einen Event-Cache in c:geo geöffnet hast, wähle:\\
-                Menü → Zu Kalender hinzufügen
-
-                Ein neuer Termin für das Event wird daraufhin in deinem Kalender erstellt.
-
-      - id: plugin-calendar-wrong-time
-        en:
-            title: Why is the appointment added without or with a wrong time?
-            content: |
-                Event cache listings on the geocaching website do only contain the date but not the time of an event.
-
-                Therefore c:geo will scan the event description and try to extract a time from it.
-                In some cases it might be wrong or we are unable to do so. In these cases you have to manually correct the entry.
-        de:
-            title: Warum wird der Termin ohne oder mit falscher Zeit hinzugefügt?
-            content: |
-                Event-Caches enthalten auf der Geocaching-Webseite nur das Datum, aber nicht die Uhrzeit des Events.
-
-                c:geo durchsucht daher die Event-Beschreibung und versucht die Uhrzeit hieraus zu extrahieren.
-                In einigen Fällen kann das zu falschen Ergebnissen führen oder es wird keine Uhrzeit gefunden. Dann musst die die Uhrzeit im Kalender-Eintrag manuell korrigieren.
-
-      - id: plugin-calendar-fails
```

Comment:
And i'm wondering if this is still necessary. I have experienced something like that myself on a phone with all standard calendar apps disabled or removed, but I assume that almost no user manages to actually have zero calendar apps. :)